### DISCUSSION TITLE
Reset

### DISCUSSION THREAD

**Discussion by samyak1409** [Belphemur/SoundSwitch#994] [Q&A] [answered]
How to reset the settings back to default?

**Comment by Belphemur** [comment, marked as answer]
Hello,
For now the easiest is to delete the folder with the settings.

1. Stop SoundSwitch
2. Delete %appdata%/SoundSwitch
3. Restart SoundSwitch

Sent from Nine
________________________________
From: Samyak Jain ***@***.***>
Sent: Monday, August 22, 2022 22:55
To: Belphemur/SoundSwitch
Cc: Subscribed
Subject: [Belphemur/SoundSwitch] Reset (Discussion #994)

How to reset the settings back to default?

**Reply by samyak1409** [reply]
Okay thank you! 😄
